### ISSUE TITLE
JUST A QUESTION!

### ISSUE THREAD

**Issue by IntelligentVoid** [elBukkit/MagicPlugin#1071]
QUESTION:

has the full set of wizard armor been implemented yet? I want the full armor set boost, but idk if the rest has been made yet.

**Comment by NathanWolf** [comment]
Yes, all 4 pieces exist. You need to add the robes example for them to show up in the RP and be craftable.

`/mexample add robes`

**Comment by IntelligentVoid** [comment]
Alrighty! Is there any other custom recipes I can add using the commands?

**Comment by IntelligentVoid** [comment]
Oh, tab complete shows me the stuff I can add

**Comment by IntelligentVoid** [comment]
I have purple wool and it doesnt show me the crafting recipe for it

**Comment by IntelligentVoid** [comment]
Ah, theres another pack

**Comment by IntelligentVoid** [comment]
It still doesnt show the item even after installing the resource pack

**Comment by IntelligentVoid** [comment]
Do I have to restart the server after applying the changes?

**Comment by NathanWolf** [comment]
You may need to restart, but shouldn't. Try the recipe even if it's not showing up in your book.

https://magic.elmakers.com/#crafting
